web/business-registry-dashboard/app/utils/account.ts

```diff
@@ -24,7 +24,7 @@ export async function hasActiveBusinessRegistryDashboardSubscription (orgId: num
     const parsedProducts = JSON.parse(products) as OrgProduct[]
 
     // Find the BUSINESS product (BRD)
-    const businessProduct = parsedProducts.find(parsedProduct => parsedProduct.code === OrgProductCodes.BUSINESS)
+    const businessProduct = parsedProducts.find(parsedProduct => parsedProduct.code === 'BUSINESS')
```

Comment:
How about extracting this into a single const/readonly string (that you can describe with a comment)?

Reply:
Looks great,